RELEASE.md

```diff
@@ -0,0 +1,61 @@
+# Release
+
+The kubernetes-base-addons repository is intended to support a minimal base set of Addons required to provide monitoring, logging, alerting, and backups for every version of Kubernetes actively supported by the upstream Kubernetes community.
+It is not intended to be tied to any specific version of Kubernetes installer (ie. Konvoy or Kommander), or Kubeaddons.
+Changes to Konvoy or Kommander to create a resource needed by an Addon should be avoided.
+
+- [Schedule](#schedule)
+- [Considerations](#considerations)
+  - [Kubernetes](#kubernetes)
+  - [Branches and Tags](#branches-and-tags)
+- [Process](#process)
+  - [Testing Release (Weekly, Thursday)](#testing-release-weekly-thursday)
+  - [Stable Release (Weekly, Wednesday)](#stable-release-weekly-wednesday)
+
+## Schedule
+
+Releases should be weekly on Wednesdays, or as needed to address CVEs.
+
+## Considerations
+
+### Kubernetes
+
+The Addons in this repo are the minimum base set of supported<sup>[1](#footnote1)</sup> Addons to be installed as a suite for any supported version of Kubernetes.
+
+Kubernetes support must be maintained for the latest general release of Kubernetes and the two prior minor releases.
+At the time of this writing, the latest Kubernetes release is 1.17.2.
+Support from this repo, at this time, should cover 1.15.x, 1.16.x, and 1.17.x.
+
+### Branches and Tags
+
+As much as possible, we will try to maintain a `master` branch that is compatible with all supported versions of Kubernetes.
```

Comment:
Doesn't this conflict with the statement "Do not use `master` for production, instead pick a supported release version." from https://github.com/mesosphere/kubernetes-base-addons/pull/119/files#diff-04c6e90faac2675aa89e2176d2eec7d8R30?

Reply:
I read this as more of a dev objective to shoot for. `master` branch will hopefully work on all k8s versions but users should still rely on the `stable-xx` tag when deploying their prod clusters.

Reply:
Right, which is why I wanted to reduce that line from a scary "warning" to just a "note".

Reply:
Got it. Thanks.